projects/kubernetes/kubernetes/1-19/KUBE_GIT_VERSION_FILE

```diff
@@ -1,6 +1,6 @@
-KUBE_GIT_COMMIT='fd5d41537aee486160ad9b5356a9d82363273721'
```

Comment:
Don't we need this change with the dev release bump?

Reply:
Moved this to https://github.com/aws/eks-distro/pull/394, which is when the dev number gets bumped. Lmk if this isn't ok

Reply:
Also I hard reset the commits back HEAD, which I know isn't the best practices for the team but I didn't want there to be any chance that something weird would happen git-wise with this PR